src/constants.ts

```diff
@@ -23,8 +23,11 @@ export const AO_CU_URL = process.env.VITE_AO_CU_URL || 'https://cu.ardrive.io';
 export const DEFAULT_ARWEAVE_PROTOCOL =
   process.env.VITE_GATEWAY_PROTOCOL ?? 'https';
 export const DEFAULT_ARWEAVE_HOST =
-  // process.env.VITE_GATEWAY_HOST ?? 'ar-io.dev';
   process.env.VITE_GATEWAY_HOST ?? 'arweave.net';
+
+export const DEFAULT_ARWEAVE_GQL_ENDPOINT =
```

Comment:
is this an available setting? may be good to add, but can be done in a separate PR

Reply:
Not a setting in the settings panel right now. Separate PR to add this sounds good to me.

Reply:
Went ahead and added this to settings